Applications/ApplicationsLib/ProjectData.cpp

```diff
@@ -57,13 +57,13 @@ ProjectData::ProjectData(BaseLib::ConfigTree const& project_config,
 {
     std::string const geometry_file = BaseLib::copyPathToFileName(
             //! \ogs_file_param{prj__geometry}
-            project_config.getConfParam<std::string>("geometry"), project_directory
+            project_config.getParameter<std::string>("geometry"), project_directory
```

Comment:
Please keep the prefix `getConf` in all methods of `ConfigTree`. You might change it to `getConfig` in order to improve naming consistency. The reason behind using `getConf` as a prefix for those methods is to be able to easily write scirpts that grep all configuration parameters from the source code. Thereby we are able to provide crosslinking between the documentation and the source code and to check (to a certain extent) if everything is documented.